x-pack/filebeat/input/lumberjack/generate_certs_test.go

```diff
@@ -151,3 +156,20 @@ func pemEncode(t testing.TB, certBytes []byte, certType string) []byte {
 
 	return pemData.Bytes()
 }
+
+func generateSubjectKeyID(pub crypto.PublicKey) []byte {
+	// SubjectKeyId generated using method 1 in RFC 7093, Section 2:
+	//   1) The keyIdentifier is composed of the leftmost 160-bits of the
+	//   SHA-256 hash of the value of the BIT STRING subjectPublicKey
+	//   (excluding the tag, length, and number of unused bits).
+	var publicKeyBytes []byte
+	switch publicKey := pub.(type) {
+	case *rsa.PublicKey:
+		publicKeyBytes = x509.MarshalPKCS1PublicKey(publicKey)
+	case *ecdsa.PublicKey:
+		//nolint:staticcheck // ignore SA101, warning for deprecated method as no alternative is present
```

Comment:
```suggestion
		//nolint:staticcheck // ignore SA1019, warning for deprecated method as no alternative is present
```
Maybe also at server_test.go:73:2.